README_new.md

```diff
@@ -0,0 +1,132 @@
+<p align="center">
+<img src="logo.png">
+</p>
+
+
+babylon health
+==================================
+
+At babylon, we firmly believe that **transparency** and **flexibility** are core values that should be present in everything we do, and hiring the best person should reflect that belief.
+
+That's why we in the iOS team have open-sourced our recruitment process, for a fairer assessment process and so that candidates know exactly what to expect, all the way to the [questions we may ask during a face-to-face interview](TODO).
```

Comment:
Starting a new paragraph with `That's why` feels very odd. Where's the missing part? I propose on merging the first two paragraphs.

Reply:
That's why we in the iOS team have open-sourced our recruitment process. This will hopefully result to a fairer assessment process with the candidates knowing exactly what to expect (all the way to the [questions we may ask during a face-to-face interview](TODO)).

Reply:
Is this better?

>At babylon, we firmly believe that **transparency** is a core value that should be present in everything we do, and hiring the best person should reflect that belief. That's why we in the iOS team have open-sourced our recruitment process, for a fairer assessment process in which candidates know exactly what to expect, all the way to the [questions we may ask during a face-to-face interview](TODO).